uint8 frame 
float32 vx
float32 vy
float32 vz
float32 yawRate
---
bool result 
